<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB Slider</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10">
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxRed">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,6">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spinBoxGreen">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QScrollBar" name="verticalScrollBar_2">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,1,6">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QScrollBar" name="verticalScrollBar_3">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spinBoxBlue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>spinBoxRed</zorder>
  <zorder>verticalScrollBar</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder></zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>graphicsView</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalScrollBar_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
